cpp/src/Slice/Parser.cpp

```diff
@@ -2404,6 +2370,42 @@ Slice::Container::hasOnlyClassDecls() const
     return true;
 }
 
+bool
+Slice::Container::hasOnlyInterfaces() const
+{
+    for(ContainedList::const_iterator p = _contents.begin(); p != _contents.end(); ++p)
```

Comment:
Any reason not to use a `for(const auto& p : _contents` here?

Reply:
Fixed.